test/parallel/test-assert.js

```diff
@@ -726,6 +726,49 @@ assert.throws(
              'assert.ok(null)\n'
   }
 );
+assert.throws(
+  () => {
+    try { assert.ok(0);   // eslint-disable-line no-useless-catch, brace-style
```

Comment:
Can you add a comment asking not to reformat and why? Otherwise someone might be tempted to simplify the test case.

Reply:
@aduh95 
Thank you for your feedback. Right, I will add comments asking not to reformat.